site/docs/get-started/OneClickDemo.md

```diff
@@ -0,0 +1,45 @@
+# One Click Demo Guide
+
+this guide contains a script that you can run with a single command to see a demo of Fybrik in action.  
+
+this demo demonstrates the following sequence -  
+
+1. A **data operator** boots up Fybrik and registers a data asset of financial data to Fybrik
```

Comment:
A **data operator** registers a data asset of financial data to Fybrik

Reply:
Fixed